sda-svc/templates/finalize-deploy.yaml

```diff
@@ -24,8 +24,8 @@ spec:
         role: finalize
         release: {{ .Release.Name }}
       annotations:
-        checksum/secret: {{ include (print $.Template.BasePath "/shared-secrets.yaml") . | sha256sum }}
```

Comment:
I'm not sure I understand the logic behind this move.

Reply:
It seems that github is a bit puzzled. I dont really see any changes...